src/Features/Core/Portable/Completion/CompletionItemFlags.cs

```diff
@@ -29,12 +30,14 @@ internal enum CompletionItemFlags
         CachedAndExpanded = Cached | Expanded,
     }
 
-    internal static class CompletionItemFlagsExtensions
+    internal static class CompletionItemFlagsHelper
     {
-        public static bool IsCached(this CompletionItemFlags flags)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool IsCached(CompletionItemFlags flags)
             => (flags & CompletionItemFlags.Cached) != 0;
 
-        public static bool IsExpanded(this CompletionItemFlags flags)
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public static bool IsExpanded(CompletionItemFlags flags)
```

Comment:
what was the benefit of this change?  (Also, it feels like in extension method form, it would still be ok).

Reply:
This stood out to me as well. The changes related to this helper type seem unnecessary.

Reply:
No, I was just trying to see if it makes any difference (it didn't). Reverted